controller/monstring.controller.php

```diff
@@ -62,7 +62,7 @@
     // Reload for å få med alle endringer
     $arrangement = new Arrangement($arrangement->getId());
 } else {
-    if ($arrangement->erFerdig()) {
+    if ($arrangement->erFerdig() && wp_get_current_user()->user_login != 'UKM Norge') {
```

Comment:
bruk heller `is_super_admin()` her (eller `is_superadmin()`) - da funker det for alle superadmin-brukere